quinn-proto/src/connection/mod.rs

```diff
@@ -814,18 +814,24 @@ where
     ///
     /// The return value if `Ok` contains the bytes and their offset in the stream.
     pub fn read_unordered(&mut self, id: StreamId) -> Result<Option<(Bytes, u64)>, ReadError> {
-        Ok(self.streams.read_unordered(id)?.map(|(buf, offset, more)| {
-            self.add_read_credits(id, more);
-            (buf, offset)
-        }))
+        Ok(self
+            .streams
+            .read_unordered(id)?
+            .map(|(buf, offset, max_stream_data_update)| {
```

Comment:
I just rename the `more`s here for a bit more clarity on what the purpose of this parameter is

Reply:
Could you please pull that out into a separate commit? I can suggest some git incantations if that helps.

Reply:
I could. Just felt it helps together with this change, and is a rather tiny rename

Reply:
I find it harder to review this change if it's mixed in with the rename, so please split it.

Reply:
Done in https://github.com/quinn-rs/quinn/pull/881